webservices/args.py

```diff
@@ -756,7 +756,7 @@ def _deserialize(self, value, attr, data):
 
 # Audit Category endpoint
 Category = {
-    'category_id': fields.List(fields.Int, description=docs.CATEGORY),
+    'category_id': fields.List(fields.Str, description=docs.CATEGORY),
     'category_name': fields.List(fields.Str, description=docs.CATEGORY),
     'tier': fields.List(fields.Int, description=docs.CATEGORY),
```

Comment:
One thing to double-check: I'm unable to "read" what I'm looking at, so I'm not sure what displays and what's just how it's built but doesn't show, but I'd like to make sure we aren't displaying the term "Tier" on the API front end since we've moved to the terms Category and Subcategory instead of the old Tier 1 and Tier 2, which proved confusing to people. I see the word `tier` here, which caught my eye

Reply:
We have removed "tier" from endpoints.
also changed "Category" to "Primary Category".